task-list/config/routes.rb

```diff
@@ -1,10 +1,16 @@
 Rails.application.routes.draw do
   root to: 'tasks#index'
-  get 'tasks/new' => 'tasks#new'
   get 'tasks/' => 'tasks#index'
+  get 'tasks/new' => 'tasks#new'
   get 'tasks/:id' => 'tasks#show'
   post 'tasks/' => 'tasks#create'
+  get 'tasks/:id' => 'tasks#show'
   delete 'tasks/:id' => 'tasks#destroy'
+  get 'tasks/:id/edit' => 'tasks#edit'
+  patch 'tasks/:id' => 'tasks#update',  as: :task
```

Comment:
Later this week we will be talking about "named routes" (what you get when you use the `as: :name` bit from this line), and why they're so powerful and convenient. However until then, I think it would be best to avoid using them at all.